detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedPrivateMember.kt

```diff
@@ -317,7 +317,11 @@ private class UnusedPropertyVisitor(allowedNames: Regex) : UnusedMemberVisitor(a
     private fun KtProperty.isMemberOrTopLevel() = isMember || isTopLevel
 
     override fun visitReferenceExpression(expression: KtReferenceExpression) {
-        nameAccesses.add(expression.text)
+        if (expression.text.startsWith("`") && expression.text.endsWith("`")) {
+            nameAccesses.add(expression.text.removePrefix("`").removeSuffix("`"))
+        } else {
+            nameAccesses.add(expression.text)
+        }
```

Comment:
```suggestion
        nameAccesses.add(expression.text.removeSurrounding("`"))        
```

I know I was wrong on how this function is used but in this particular case it really does what it is supposed to do.

Reply:
TIL :O